activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Add `Cache#read_counter` and `Cache#write_counter`
+
+    ```ruby
+    Rails.cache.write_counter("foo", 1)
+    Rails.cache.read_counter("foo") # => 1
+    Rails.cache.increment("foo")
```

Comment:
Doesn't `increment` also return the new value, or keeping that out for readability?

Reply:
Yes, and yes.
But happy to change if you think it's confusing.